packages/forms/src/fields/BooleanField.js

```diff
@@ -39,10 +40,16 @@ function BooleanField(props) {
 	const onChangeHandler = () => {
 		onChange(!formData, uiOptions);
 	};
+	const onBlurHandler = () => {
```

Comment:
I'm pretty sure, you can add (event) => {

   onBlur(event, { id: odSchema.$id, formData }) would be a better API

For me event handler API is always (event, data)